contracts/solidity/set_escrow.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+if [ -f ./contracts/solidity/.env ]; then
+    echo "Sourcing .env file..."
+    source ./contracts/solidity/.env
+else
+    echo "Error: .env file not found!"
+    exit 1
+fi
+if [ -f ./contracts/cairo/.env ]; then
+    echo "Sourcing .env file..."
+    source ./contracts/cairo/.env
+else
+    echo "Error: .env file not found!"
+    exit 1
+fi
```

Comment:
set-escrow.sh needs $ETH_CONTRACT_ADDR , stored in cairo/.env because it is used by cairo/deploy.sh 

I thought about adding this variable to solidity/.env , but it would make this variable repeated in both .env, situation that would enable a dangerous bug case if both variables are different (it would seem to run OK but it would end up misconfigured)

Reply:
great thanks